flare_emu.py

```diff
@@ -461,6 +461,13 @@ def getEmuWideString(self, addr):
             addr += 2
         return out
 
+    # returns a <size> string of bytes read from <addr>
+    def getEmuBytes(self, addr, size):
+        out = ""
+        for i in range(addr, addr+size):
+            out += self.uc.mem_read(i, 1)
+        return out

```

Comment:
How about `return self.uc.mem_read(addr, size)` instead of the loop?

Reply:
It makes a lot more sense indeed!